modules/claims_api/app/workers/claims_api/claim_uploader.rb

```diff
@@ -23,6 +24,7 @@ def perform(uuid)
         uploader.retrieve_from_store!(claim_object.file_data['filename'])
         file_body = uploader.read
         service(auth_headers).upload(file_body, claim_upload_document(claim_object))
+        ClaimsApi::Logger.log('526', claim_id: auto_claim.id, attachment_id: uuid)
```

Comment:
We need to move this up just before the `service()` call.
That way, even if the service call blows up, we'll always get a log from right before the moment when it happened.

Reply:
Doesn't the service do the upload, I don't want to say that it's been uploaded if it hasn't yet; should I `begin service() ensure log() end` block?

Reply:
> I don't want to say that it's been uploaded if it hasn't yet

But you're not doing that.  Right now you're just generating a log that will say, 
`ClaimsApi :: Claim ID: 123ABC :: Attachment ID: 987XYZ`

It doesn't mention anything about uploading anything.